builds/posix/Makefile.in.examples

```diff
@@ -91,7 +99,7 @@ EMPLOYEE_DB=	$(EXAMPLES_DEST)/employee.fdb
 FINAL_EMPDB=	$(EXAMPLES_FB)/empbuild/employee.fdb
 INTLEMP_DB=	$(EXAMPLES_DEST)/intlemp.fdb
 
-EXTAUTH_PLUGIN=	$(EXAMPLES_FB)/prebuilt/libfbSampleExtAuth.$(SHRLIB_EXT)
+EXTAUTH_PLUGIN=	$(EXAMPLES_FB)/prebuilt/$(LIB_PREFIX)fbSampleExtAuth.$(SHRLIB_EXT)
 
```

Comment:
Why build plugin in LIB_PREFIX?

Reply:
This is to mimic MSVC build, where the example dlls doesn't have `lib` prefix